include/exadg/postprocessor/kinetic_energy_spectrum.cpp

```diff
@@ -596,17 +596,17 @@ KineticEnergySpectrumCalculator<dim, Number>::setup(
   data        = data_in;
   dof_handler = &dof_handler_in;
 
-  AssertThrow(
-    dof_handler->get_triangulation().all_reference_cells_are_hyper_cube(),
-    dealii::ExcMessage(
-      "This postprocessing utility is only available for meshes consisting of hypercube elements."));
-
   clear_files = data.clear_file;
 
   time_control.setup(data_in.time_control_data);
 
   if(data_in.time_control_data.is_active)
```

Comment:
This if-statement uses `data_in`, so I wonder whether we should actually shift everything before this if-statement inside the if-statement.

Reply:
Then, the assert on the dof_handler/triangulation would again appear next to the assignment of the dof-handler.

Reply:
Yep!